cmd/gator/sync/test/test.go

```diff
@@ -0,0 +1,70 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	cmdutils "github.com/open-policy-agent/gatekeeper/v3/cmd/gator/util"
+	"github.com/open-policy-agent/gatekeeper/v3/pkg/gator/reader"
+	"github.com/open-policy-agent/gatekeeper/v3/pkg/gator/sync/test"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "test",
+	Short: "Test that the provided SyncSet(s) and/or Config contain the GVKs required by the input templates.",
+	Run:   run,
+}
+
+var (
+	flagFilenames       []string
+	flagImages          []string
+	flagOmitGVKManifest bool
+)
+
+const (
+	flagNameFilename = "filename"
+	flagNameImage    = "image"
+	flagNameForce    = "omit-gvk-manifest"
```

Comment:
Will do--thinking now about shorthand ux, -f is obviously the most common for force, but I suppose I should be consistent with the rest of gator in using -f for the files piped in...

Reply:
Well, -f is used for output format in gator expand and for the filenames in gator test--filenames are denoted by -n in gator expand and passed in as an arg to gator verify, so it seems consistency isn't super high priority and we could alternatively use -f for force

Reply:
`gator` is in alpha, so you could open an issue and talk about this idea with the references you describe.  In the long term it would be good to have consistency across the commands should we ever get a chance to re-write `gator`.

Reply:
FYI: `gator` was promoted to beta in 3.11: https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0

Reply:
Created!